cartography/cli.py

```diff
@@ -314,6 +353,16 @@ def main(self, argv):
             # No need to store the returned value; we're using this for input validation.
             parse_and_validate_aws_requested_syncs(config.aws_requested_syncs)
 
+        # Okta config
```

Comment:
I think you copy pasted this by accident

Reply:
:)

i have updated it now.